packages/fxa-auth-server/lib/routes/utils/subscriptions.js

```diff
@@ -29,9 +31,12 @@ const SubscriptionUtils = (module.exports = {
   determineClientVisibleSubscriptionCapabilities: async function(
     stripeHelper,
     uid,
-    client_id,
+    clientIdRaw,
     email
   ) {
+    const clientId = Buffer.isBuffer(clientIdRaw)
+      ? hex(clientIdRaw).toLowerCase()
+      : clientIdRaw;
```

Comment:
FWIW I believe the `hex` helper does this `isBuffer` check internally already, so you shouldn't need to do it here.

Reply:
Ah, yup, looks like that's right. Though we *do* need to test whether clientIdRaw ends up null here